config.toml

```diff
@@ -1,3 +1,4 @@
+version = 2
```

Comment:
This TOML file is not for containerd. So we must not have `version = 2` here.

Reply:
Ok, got it.. Just for my understanding, what's the pupose of this file ?

Reply:
That is for https://gohugo.io/.

Reply:
ah, got it. Thanks @kzys